toplevel/opttopdirs.ml

```diff
@@ -113,7 +113,7 @@ type 'a printer_type_old = 'a -> unit
 let match_printer_type ppf desc typename =
   let printer_type =
     try
-      Env.lookup_type (Ldot(Lident "Topdirs", typename)) !toplevel_env
+      Env.lookup_type (Ldot(Lident "Opttopdirs", typename)) !toplevel_env
```

Comment:
Yes, it fixes a problem I had while testing that `ocamlnat` was still working after my changes.